SDL/LST.cc

```diff
@@ -74,7 +76,8 @@ void SDL::LST<SDL::Acc>::loadAndFillES(alpaka::QueueCpuBlocking& queue, struct m
                            pLStoLayer);
 }
 
-void SDL::LST<SDL::Acc>::run(SDL::QueueAcc& queue,
+void SDL::LST<SDL::Acc>::run(SDL::Dev& devAccIn,
+                             SDL::QueueAcc& queue,
```

Comment:
does it make sense to have a queue and a device separately? Isn't a queue uniquely attached to a device?

Reply:
That's a good point. I'll tidy things up.